src/test/java/com/wikia/webdriver/elements/mercury/components/discussions/common/category/CategoriesFieldset.java

```diff
@@ -0,0 +1,189 @@
+package com.wikia.webdriver.elements.mercury.components.discussions.common.category;
+
+import com.google.common.collect.Iterables;
+import com.wikia.webdriver.pageobjectsfactory.pageobject.WikiBasePageObject;
+import org.openqa.selenium.By;
+import org.openqa.selenium.WebElement;
+import org.openqa.selenium.support.FindBy;
+
+import javax.annotation.CheckForNull;
+import java.util.List;
+import java.util.function.Function;
+
+public class CategoriesFieldset extends WikiBasePageObject {
+
+  private static final String GENERAL_CATEGORY_NAME = "General";
+
+  @FindBy(className = "discussion-categories")
+  private WebElement fieldset;
+
+  @FindBy(css = ".discussion-categories > legend.mobile-hidden")
+  private WebElement label;
+
+  @FindBy(className = "discussion-categories-edit-link")
+  private WebElement editLink;
+
+  @FindBy(css = ".discussion-categories .discussion-categories-list > li")
+  private List<WebElement> categories;
+
+  @FindBy(className = "discussion-categories-edit")
+  private WebElement categoriesEdit;
+
+  @FindBy(className = "discussion-category-all")
+  private WebElement editableCategoryAll;
+
+  @FindBy(css = ".discussion-categories-list .sortable-item")
+  private List<WebElement> editableCategories;
+
+  @FindBy(className = "discussion-categories-edit-add-link")
+  private WebElement addCategoryLink;
+
+  @FindBy(css = ".discussion-categories-edit button.submit")
+  private WebElement approveChangesButton;
+
+  @FindBy(className = "categories-edit-info-message")
+  private WebElement infoMessage;
+
+  public boolean canEdit() {
+    WebElement svg = fieldset.findElement(By.tagName("use"));
+    return svg.getAttribute("xlink:href").equals("#pencil");
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Move the "#pencil" string literal on the left side of this string comparison. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](http://sonar-s1/coding_rules#rule_key=squid%3AS1132)